src/ontology/cl-edit.owl

```diff
@@ -24989,6 +24989,9 @@ EquivalentClasses(obo:CL_1000217 ObjectIntersectionOf(obo:CL_0000138 ObjectSomeV
 
 # Class: obo:CL_1000222 (stomach neuroendocrine cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1196/annals.1294.001") obo:IAO_0000115 obo:CL_1000222 "A specialised neuroendocrine cell located in the gastric mucosa that regulates digestive processes including acid secretion and motility. This cell stores hormones in large dense core vesicles and synaptic-like microvesicles.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1196/annals.1294.001") oboInOwl:hasBroadSynonym obo:CL_1000222 "DES cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1196/annals.1294.001") oboInOwl:hasBroadSynonym obo:CL_1000222 "diffuse endocrine system cell")
 AnnotationAssertion(rdfs:label obo:CL_1000222 "stomach neuroendocrine cell")
 EquivalentClasses(obo:CL_1000222 ObjectIntersectionOf(obo:CL_0000165 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000945)))
 
```

Comment:
Consider adding 'capable of' some 'acid secretion'

Reply:
Added